### PR TITLE
Fix course exporter.

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: exporter/course_export.py [hunk 1]
```diff
@@ -4,11 +4,10 @@
 Export course data.
 
 Usage:
-  course-exporter [options] <config> <org-config> [--env=<environment>...] [--course=<course>...] [--task=<task>...]
```
Comment (azafty468) [inline exporter/course_export.py:7]: So I have a question.  How is an individual org specified?  Or is this just a well named class and it only exports one course?

Comment (HassanJaveed84) [reply exporter/course_export.py:7]: So this piece of code is supposed to export for a list of courses without any org involvement. I'm not sure why we added org-config parameter here. 
@brianhw can you confirm this ?

Comment (azafty468) [reply exporter/course_export.py:7]: Ha, if it runs without the org is sounds like it's not necessary

**Review by azafty468 — APPROVED** [review]

File: exporter/course_export.py (hunk 1 above)
Comment (brianhw) [reply exporter/course_export.py:7]: I suspect this was a cut-and-paste.   I expect it's not so much that we added org here as failed to remove it when modifying the regular exporter.   Maybe you could ask the person who implemented this.... : )

**Review by brianhw — APPROVED** [review]
LGTM, best I can tell.  👍

File: exporter/metrics.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def collect_elapsed_time(cls, **kwargs):
 
     metric_name = get_metric_name(cls, **kwargs)
     if not metric_name:
-        log.info('No organization for this task, not sending metrics. Available kwargs are: %s', kwargs)
+        log.info('No organization for this task, not sending metrics.')
```
Comment (HassanJaveed84) [inline exporter/metrics.py:57]: This printed out kwargs on the console, including secrets..

**Review by rao-abdul-mannan — APPROVED** [review]
LGTM

**Pull request merged**